index.src.html

```diff
@@ -503,25 +505,26 @@ <h3 id="process-header" algorithm>
 
   6.  For each |item| in |list|:
 
-      1.  If |item| has no member named "<a for="ReportTo">`max_age`</a>", or that member's
-          value is not a number, skip to the next |item|.
+      1.  If |item| has no member named "<a for="ReportTo">`max_age`</a>", or
+          that member's value is not a number, skip to the next |item|.
 
       2.  If |item| has no member named "<a>`endpoints`</a>", or that member's
           value is not an array, skip to the next |item|.
 
-      3.  Let |name| be |item|'s "<a for="ReportTo">`group`</a>" member's value if present, and
-          "`default`" otherwise.
+      3.  Let |name| be |item|'s "<a for="ReportTo">`group`</a>" member's value
+          if present, and "`default`" otherwise.
 
       4.  If there is already an <a>endpoint group</a> in |groups| whose
           {{endpoint group/name}} is |name|, skip to the next |item|.
 
       5.  Let |endpoints| be an empty list.
 
-      6.  For each |endpoint item| in the value of |item|'s "<a for="ReportTo">`endpoints`</a>"
-          member:
+      6.  For each |endpoint item| in the value of |item|'s
+          "<a for="ReportTo">`endpoints`</a>" member:
```

Comment:
You can break tags between attributes, so this would normally be wrapped as

    For each |endpoint item| in the value of |item|'s "<a
    for="ReportTo">`endpoints`</a>" member: